install/kubernetes/cilium/charts/hubble-ui/values.yaml

```diff
@@ -0,0 +1,61 @@
+# image used for hubble server
+image:
+  # repository of the docker image
+  repository: quay.io/cilium/hubble
+  # tag is the container image tag to use
+  tag: latest
+  # pullPolicy is the container image pull policy
+  pullPolicy: Always
+
+# Configuration for hubble ui
+ui:
+  enabled: false
+  image:
+    # repository of the docker image
+    repository: quay.io/cilium/hubble-ui
+    # tag is the container image tag to use
+    tag: latest
+    # pullPolicy is the container image pull policy
+    pullPolicy: Always
+  clusterDomain: cluster.local
+  replicas: 1
+  # Resource requests and limits for the 'hubble-ui' container of the 'hubble-ui' deployment, such as
+  #
+  #     resources:
+  #       limits:
+  #         cpu: 1000m
+  #         memory: 1024Mi
+  #       requests:
+  #         cpu: 100m
+  #         memory: 64Mi
+  resources: {}
+  # The priority class system-node-critical marks add-on pods as critical to the node itself.
+  # This priority class is only valid under the kube-system namespace.
+  priorityClassName: ""
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+# Resource requests and limits for the 'hubble' container of the 'hubble' daemonset, such as
+#
+#     resources:
+#       limits:
+#         cpu: 1000m
+#         memory: 1024Mi
+#       requests:
+#         cpu: 100m
+#         memory: 64Mi
+resources: {}
+# The priority class system-node-critical marks add-on pods as critical to the node itself.
+# This priority class is only valid under the kube-system namespace.
+priorityClassName: system-node-critical
```

Comment:
All of this must also be removed as well.

Reply:
There is a check on the priorityClassName on install/kubernetes/cilium/charts/hubble-ui/templates/deployments.yaml.
For now I will leave it there and see what you think.

Reply:
Ah, did not catch that. Not sure if it really makes sense to mark the ui as critical to be honest.